src/apps/search-header/search-header.tsx

```diff
@@ -98,12 +96,17 @@ const SearchHeader: React.FC = () => {
 
   // downshift prevents the default form submission event when the autosuggest
   // is open - that's why in some cases we have to simulate form sumbission
-  function manualRedirect(inputValue: string) {
-    const baseUrl = t("searchHeaderUrlText");
-    const params = inputValue;
-    if (window.top) {
-      window.top.location.href = `${baseUrl}?q=${params}`;
+  function manualRedirect(itemValue: Suggestion) {
+    const inputValue = itemToString(itemValue);
+    if (!window.top) {
+      return;
+    }
```

Comment:
Consider throwing an error if this occurs.

I do not know why `window.top` would not be set but with this approach it fails silently. This can make it hard to debug. Throwing an error may make it easier for us to debug in the future.